xdsl/dialects/smt.py

```diff
@@ -424,6 +424,20 @@ def __init__(self, *values: SSAValue):
         super().__init__(operands=[values], result_types=[])
 
 
+@irdl_op_definition
+class AssertOp(IRDLOperation):
+    """Assert that a boolean expression holds."""
+
+    name = "smt.assert"
+
+    input = operand_def(BoolType)
+
+    assembly_format = "$input attr-dict"
+
+    def __init__(self, input: SSAValue):
+        super().__init__(operands=[input], result_types=[])
```

Comment:
```suggestion
        super().__init__(operands=[input])
```
Pretty sure this is default